chromadb/test/distributed/test_sanity.py

```diff
@@ -23,6 +23,9 @@
 def test_add(
     client: ClientAPI,
 ) -> None:
+    seed = time.time()
+    random.seed(seed)
```

Comment:
should we do the same here https://github.com/chroma-core/chroma/pull/2507/files#diff-8ec7688023303502f0bce17f9ac8849df94379211b8d287d3430217f8a7548fbR82